src/main/java/com/ldtteam/structurize/util/TickedWorldOperation.java

```diff
@@ -240,7 +244,12 @@ private boolean run(final ServerWorld world)
                 {
                     final BlockPos here = new BlockPos(x, y, z);
                     final BlockState blockState = world.getBlockState(here);
-                    final ItemStack stack = BlockUtils.getItemStackFromBlockState(blockState);
+                    // can not use strict version because we are placing block
+                    ItemStack stack = BlockUtils.getItemStackFromBlockState(blockState);
```

Comment:
this itemstack check is actually super weird - not sure what it does why it's here, dump block equals check should suffice